functional-samples/tutorial.websockets/README.md

```diff
@@ -0,0 +1,12 @@
+# Chrome Extension WebSocket Example
+
+This example demonstrates how to use WebSockets in a MV3 Chrome Extension.
+
```

Comment:
Do we need to add a minimum Chrome version prereq?

Reply:
good point - done!